cloudbridge/cloud/providers/gce/services.py

```diff
@@ -417,3 +419,59 @@ def instances(self):
     @property
     def regions(self):
         return self._region_svc
+
+
+class GCENetworkService(BaseNetworkService):
+
+    def __init__(self, provider):
+        super(GCENetworkService, self).__init__(provider)
+
+    def get(self, network_id):
+        networks = self.list(filter='id eq %s' % network_id)
+        return None if len(networks) == 0 else networks[0]
+
+    def list(self, limit=None, marker=None, filter=None):
+        try:
+            response = (self.provider.gce_compute
+                                     .networks()
+                                     .list(project=self.provider.project_name,
+                                           filter=filter)
+                                     .execute())
+            networks = []
+            if 'items' in response:
+                for network in response['items']:
+                    networks.append(GCENetwork(self.provider, network))
+            return networks
+        except:
+            return []
+
+    def create(self, name):
+        try:
+            response = (self.provider.gce_compute
+                                     .networks()
+                                     .insert(project=self.provider.project_name,
+                                             body={'name': name})
+                                     .execute())
+            if 'error' in response:
+                return None
+            self.provider.wait_for_global_operation(response)
+            networks = self.list(filter='name eq %s' % name)
```

Comment:
I think we need to come up with a way to standardise the filter vocabulary, since this won't work on non-GCE clouds. However, since the filter parameter is woefully underspecified in the cloudbridge spec, until we come up with a generalised solution, I propose we leave this as is for now.